examples/step-62/step-62.cc

```diff
@@ -0,0 +1,1467 @@
+            solution_names.emplace_back("displacement_z");
+          }
+        std::vector<DataComponentInterpretation::DataComponentInterpretation>
+          interpretation(dim, DataComponentInterpretation::component_is_scalar);
+
+        DataOut<dim> data_out;
+        data_out.add_data_vector(dof_handler,
+                                 locally_relevant_solution,
+                                 solution_names,
+                                 interpretation);
+        Vector<float> subdomain(triangulation.n_active_cells());
+        for (unsigned int i = 0; i < subdomain.size(); ++i)
+          subdomain(i) = triangulation.locally_owned_subdomain();
+        data_out.add_data_vector(subdomain, "subdomain");
+
+        std::vector<Vector<double>> force(
+          dim, Vector<double>(triangulation.n_active_cells()));
+        std::vector<Vector<double>> pml(
+          dim, Vector<double>(triangulation.n_active_cells()));
+        Vector<double> rho(triangulation.n_active_cells());
+
+        for (auto cell : triangulation.active_cell_iterators())
+          {
+            if (cell->is_locally_owned())
+              {
+                for (unsigned int dim_idx = 0; dim_idx < dim; ++dim_idx)
+                  {
+                    force[dim_idx](cell->active_cell_index()) =
+                      parameters.right_hand_side.value(cell->center(), dim_idx);
+                    pml[dim_idx](cell->active_cell_index()) =
+                      parameters.pml.value(cell->center(), dim_idx).imag();
+                  }
+                rho(cell->active_cell_index()) =
+                  parameters.rho.value(cell->center());
+              }
+            // And on the cells that we are not interested in, set the
+            // respective value to a bogus value in order to make sure that if
+            // we were somehow wrong about our assumption we would find out by
+            // looking at the graphical output:
+            else
+              {
+                for (unsigned int dim_idx = 0; dim_idx < dim; ++dim_idx)
+                  {
+                    force[dim_idx](cell->active_cell_index()) =
+                      parameters.right_hand_side.value(cell->center(), dim_idx);
+                    pml[dim_idx](cell->active_cell_index()) = -1e+20;
+                  }
+              }
+          }
+
+        for (unsigned int dim_idx = 0; dim_idx < dim; ++dim_idx)
+          {
+            data_out.add_data_vector(force[dim_idx],
+                                     "force_" + std::to_string(dim_idx));
+            data_out.add_data_vector(pml[dim_idx],
+                                     "pml_" + std::to_string(dim_idx));
+          }
+        data_out.add_data_vector(rho, "rho");
+
+        data_out.build_patches();
+
+        unsigned int      nb_number_positions;
+        std::stringstream frequency_idx_stream;
+        nb_number_positions =
+          ((unsigned int)std::log10(parameters.nb_frequency_points)) + 1;
```

Comment:
just move the assignment into the initialization of the variable. then it can also be `const`